content/dart/concepts/list/terms/join/join.md

```diff
@@ -0,0 +1,110 @@
+---
+Title: 'The Title' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'A brief description.' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'A subject name'
+  - 'A second subject name'
+  - 'An nth subject name'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'A tag'
+  - 'A second tag'
+  - 'An nth tag'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-example-course'
+  - 'paths/example-path'
+---

```

Comment:
Please fill in the details here. You can find the listed [subjects](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md), [tags](https://github.com/Codecademy/docs/blob/main/documentation/tags.md), and [catalog content](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md). Check out the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) if you want.